internal/pkg/cli/deploy/workload.go

```diff
@@ -367,6 +376,11 @@ func (d *workloadDeployer) uploadContainerImages(out *UploadArtifactsOutput) err
 	}
 	out.ImageDigests = make(map[string]ContainerImageIdentifier, len(buildArgsPerContainer))
 	for name, buildArgs := range buildArgsPerContainer {
+		buildArgsList, err := buildArgs.GenerateDockerBuildArgs(dockerengine.New(exec.NewCmd()))
+		if err != nil {
+			return fmt.Errorf("generate docker build args: %w", err)
+		}
+		log.Infof(fmt.Sprintf("Building your container image: docker %s\n", strings.Join(buildArgsList, " ")))
```

Comment:
i know we are getting rid of this in the next PR anyways, but we don't need the `fmt.Sprintf` as `log.Infof` already does formatting